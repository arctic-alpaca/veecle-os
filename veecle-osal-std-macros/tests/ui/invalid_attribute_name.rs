#[veecle_osal_std_macros::main(invalid = "debug")]
async fn main() {}
