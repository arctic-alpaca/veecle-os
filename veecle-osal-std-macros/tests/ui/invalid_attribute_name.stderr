error: Unknown field: `invalid`. Available values: `telemetry`
 --> tests/ui/invalid_attribute_name.rs:1:32
  |
1 | #[veecle_osal_std_macros::main(invalid = "debug")]
  |                                ^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/invalid_attribute_name.rs:2:19
  |
2 | async fn main() {}
  |                   ^ consider adding a `main` function to `$DIR/tests/ui/invalid_attribute_name.rs`
